Messaging/Two Factor Authentication via SMS - Node.js/README.md

```diff
@@ -0,0 +1,40 @@
+# Two Factor Authentication via SMS in Node.js
+
+## Overview
+Two Factor Authentication (2FA) can provide your users effective protection against many security threats that target user passwords and accounts. This application will generate a one-time password that is sent to the recipient's phone number via SMS. Application developers can enable two-factor authentication for their users with ease and without making any changes to the already existing application logic or database structure!
+
+
+## Required Resources
+
+View the [full walkthrough](https://developer.signalwire.com/apis/docs/two-factor-authentication) on our developer portal!
```

Comment:
Could we also add the usual "Read the full guide" link/emoji?

Like here: https://github.com/signalwire/guides/tree/main/Video/Embed%20Video%20Rooms%20-%20Videoconferencing%20Widget